UnityProject/Assets/Scripts/Core/Editor/EditorDrawPreview.cs

```diff
@@ -102,6 +102,7 @@ static EditorDrawPreview()
 
 	private static void DrawProjectItem(string guid, Rect selectionRect)
 	{
+		if (EditorPrefs.GetBool("editorPreviewsDisable"), false) return;
```

Comment:
I might have miss placed that ``false`` if you committed it directly from the suggestion